### PR TITLE
Add a new configurable option to the windows installer for static mode

### LINKED ISSUE #5554: Let the windows installer create the service with the -config.expand-env parameter
### Request

It would be nice if it's possible to run the windows installer of the grafana agent with a parameter to create the windows service with the -config.expand-env parameter

### Use case

installing grafana agent on windows

### PR DESCRIPTION
**Pull request by erikbaranowski**
#### PR Description

Closes #5554

#### PR Checklist

- [x] CHANGELOG.md updated
- [x] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

**Comment by mattdurham** [comment]
Can we add a blurb on https://github.com/grafana/agent/blob/main/docs/sources/static/set-up/install/install-agent-on-windows.md about passing in the silent var?

File: docs/sources/static/set-up/install/install-agent-on-windows.md [hunk 1]
```diff
@@ -30,7 +30,11 @@ To do a standard graphical install of Grafana Agent on Windows, perform the foll
 1. Double-click on `grafana-agent-installer.exe` to install Grafana Agent.
 
    Grafana Agent is installed into the default directory `C:\Program Files\Grafana Agent`.
-   The [windows_exporter integration](/docs/agent/latest/static/configuration/integrations/windows-exporter-config) can be enabled with all default windows_exporter options.
+   
+   The following options are available:
+   
+   - The [windows_exporter integration][windows_exporter_config] can be enabled with all default windows_exporter options.
```
Comment (erikbaranowski) [inline docs/sources/static/set-up/install/install-agent-on-windows.md:36]: This was hardcoded to /latest/ but should be corrected now

**Review by mattdurham — APPROVED** [review]
Looks good to me

File: docs/sources/static/set-up/install/install-agent-on-windows.md [hunk 2]
```diff
@@ -78,6 +82,24 @@ If you are using `remote_write` you must enable Windows Exporter and set the glo
 
    If you are using Powershell, make sure you use triple quotes `"""http://example.com"""` around the URL parameter.
 
+## Silent install with `-config.expand_env`
+
+It is possible to enable [-config.expand-env][flags] during a silent install.
```
Comment (clayton-cornell) [inline docs/sources/static/set-up/install/install-agent-on-windows.md:87]:
```suggestion
You can enable [-config.expand-env][flags] during a silent install.
```

**Review by clayton-cornell — COMMENTED** [review]
Small change to the wording

**Review by clayton-cornell — APPROVED** [review]

**Pull request merged**